.github/workflows/memcheck.yml

```diff
@@ -83,16 +83,8 @@ jobs:
         shell: bash
         run: echo "The SAFE_PEERS variable has been set to $SAFE_PEERS"
 
-      - name: Start a faucet client to claim genesis
-        run: cargo run --bin faucet --release -- claim-genesis
-        env:
-          SN_LOG: "all"
-        timeout-minutes: 2
-
       - name: Create and fund a wallet to pay for files storage
-        run: |
-          cargo run --bin faucet --release -- send 500 $(cargo run --bin safe --release -- wallet address | tail -n 1) | tail -n 1 > dbc_hex
-          cat dbc_hex | cargo run --bin safe --release -- wallet deposit --stdin
+        run: cargo run --bin safe --release -- wallet get-faucet
```

Comment:
get faucet will only get 10DBC, is that enough?

Reply:
for now yeh, storage is 1 nano fixed price atm